OrbitGl/FunctionsDataView.cpp

```diff
@@ -248,36 +261,42 @@ void FunctionsDataView::OnContextMenu(const std::string& a_Action,
                                       const std::vector<int>& a_ItemIndices) {
   if (a_Action == MENU_ACTION_SELECT) {
     for (int i : a_ItemIndices) {
-      Function& function = GetFunction(i);
-      function.Select();
+      Function* function = GetFunction(i);
```

Comment:
Maybe populate a list of valid Function* once at the top of the function and iterate without checking for nullptr inside the if statements?

Reply:
Good point, done.